README.md

```diff
@@ -116,24 +121,17 @@ BuildKit is used by the following projects:
 :information_source: For Kubernetes deployments, see [`examples/kubernetes`](./examples/kubernetes).
 
 BuildKit is composed of the `buildkitd` daemon and the `buildctl` client.
-While the `buildctl` client is available for Linux, macOS, and Windows, the `buildkitd` daemon is only available for Linux currently.
-
-The `buildkitd` daemon requires the following components to be installed:
--   [runc](https://github.com/opencontainers/runc) or [crun](https://github.com/containers/crun)
--   [containerd](https://github.com/containerd/containerd) (if you want to use containerd worker)
+While the `buildctl` client is available for Linux, macOS, and Windows, the `buildkitd` daemon is only available for Linux and *Windows currently.
 
 The latest binaries of BuildKit are available [here](https://github.com/moby/buildkit/releases) for Linux, macOS, and Windows.
 
-[Homebrew package](https://formulae.brew.sh/formula/buildkit) (unofficial) is available for macOS.
-```console
-$ brew install buildkit
-```
-
-To build BuildKit from source, see [`.github/CONTRIBUTING.md`](./.github/CONTRIBUTING.md).
 
-For a `buildctl` reference, see [this document](./docs/reference/buildctl.md).
+### Linux Setup
+The `buildkitd` daemon requires the following components to be installed:
+-   [runc](https://github.com/opencontainers/runc) or [crun](https://github.com/containers/crun)
+-   [containerd](https://github.com/containerd/containerd) (if you want to use containerd worker)
 
-### Starting the `buildkitd` daemon
+#### Starting the `buildkitd` daemon
```

Comment:
I think we can remove this title to keep TOC consistent across OSes

Reply:
I can just make it bold, right?